ngx-datetimepicker/src/app/date.component/date.component.html

```diff
@@ -1,66 +1,28 @@
-          <button class="btn__calendar btn__calendar__year">2017</button>
-        </div>
-        <div class="calendar--days-of-week">
-          <span class="calendar--day-of-week">Su</span>
-          <span class="calendar--day-of-week">Mo</span>
-          <span class="calendar--day-of-week">Tu</span>
-          <span class="calendar--day-of-week">We</span>
-          <span class="calendar--day-of-week">Th</span>
-          <span class="calendar--day-of-week">Fr</span>
-          <span class="calendar--day-of-week">Sa</span>
-        </div>
-        <div class="calendar--day-select">
-          <span class="calendar--day calendar--day__muted">29</span>
-          <span class="calendar--day calendar--day__muted">30</span>
-          <span class="calendar--day calendar--day__muted">31</span>
-          <span class="calendar--day">1</span>
-          <span class="calendar--day">2</span>
-          <span class="calendar--day">3</span>
-          <span class="calendar--day">4</span>
-          <span class="calendar--day">5</span>
-          <span class="calendar--day">6</span>
-          <span class="calendar--day">7</span>
-          <span class="calendar--day">8</span>
-          <span class="calendar--day">9</span>
-          <span class="calendar--day calendar--day__selected">10</span>
-          <span class="calendar--day">11</span>
-          <span class="calendar--day">12</span>
-          <span class="calendar--day">13</span>
-          <span class="calendar--day">14</span>
-          <span class="calendar--day">15</span>
-          <span class="calendar--day">16</span>
-          <span class="calendar--day">17</span>
-          <span class="calendar--day">18</span>
-          <span class="calendar--day">19</span>
-          <span class="calendar--day">20</span>
-          <span class="calendar--day">21</span>
-          <span class="calendar--day">22</span>
-          <span class="calendar--day">23</span>
-          <span class="calendar--day">24</span>
-          <span class="calendar--day">25</span>
-          <span class="calendar--day">26</span>
-          <span class="calendar--day">27</span>
-          <span class="calendar--day">28</span>
-          <span class="calendar--day calendar--day__muted">1</span>
-          <span class="calendar--day calendar--day__muted">2</span>
-          <span class="calendar--day calendar--day__muted">3</span>
-          <span class="calendar--day calendar--day__muted">4</span>
-        </div>
-        <div class="calendar--footer">
-          <button class="btn__calendar btn__calendar__now">Now</button>
-          <button class="btn__calendar btn__calendar__done">Done</button>
-        </div>
-      </div>
+<div class="container">
+	<div class="calendar">
+		<div class="calendar--month-and-year">
+			<button class="btn__calendar btn__calendar__month">{{ selectedMonthText }}</button>
+			<button class="btn__calendar btn__calendar__year">{{ selectedYear }}</button>
```

Comment:
I have a selectedMonth property that returns 2 , 4 etc so I figured text in this case makes sense since it's 'May', 'March' etc

Does that make sense?